libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -342,6 +370,14 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
         return row.value instanceof FieldGroup ? row.value.fields : row.value;
     }
 
+    buildComponentCssClass(): string[] {
+        return [
+            'fd-container',
+            this.cozy ? 'fd-form-layout-grid-container' : '',
```

Comment:
3. Switch `this.cozy` to `!this.compact`
```suggestion
            !this.compact ? 'fd-form-layout-grid-container' : '',
```

_______
Hmm, probably this line is redundant since the template handle this class